reactor-netty-core/src/main/java/reactor/netty/tcp/SslProvider.java

```diff
@@ -94,6 +94,12 @@ public static SslProvider addHandlerConfigurator(
 		return new SslProvider(provider, handlerConfigurator);
 	}
 
+	/**
+	 * @deprecated as of 1.0.6. Prefer {@link SslProvider.SslContextSpec#sslContext(ProtocolSslContextSpec)},
+	 * where the default configuration is applied before any other custom configuration.
+	 * This will be removed in version 1.2.0.
```

Comment:
should it state "at the earliest"? in the deprecation policy, we state that such versions are only a "minimal" date and that deprecated methods can be kept around for longer than that if they don't hurt too much (with the caveat that past the stated version, the method can be removed without particular notice in any patch release)

Reply:
well this is something that we want to remove as soon as possible, isn't it?